numpy/random/_generator.pyx

```diff
@@ -635,26 +635,26 @@ cdef class Generator:
 
         Parameters
         ----------
-        a : 1-D array-like or int
+        a : {array_like, int}
             If an ndarray, a random sample is generated from its elements.
-            If an int, the random sample is generated as if a were np.arange(a)
+            If an int, the random sample is generated as if a is np.arange(a).
```

Comment:
_Were_ was correctly subjunctive.

Reply:
Seems ok either way, could also do: the result is the same as for np.arange(a). Subjunktive is not really simple english ;), but then here it is probably fairly clear either way.

Reply:
I shortened it to get it on 1 line the . which was missing.  I have rewritten it in simpler language now.